### DISCUSSION TITLE
Scraping Recipes Using Node.js, Pipedream, and JSON-LD

### DISCUSSION THREAD

**Discussion by giscus** [cfjedimaster/raymondcamden2023#22] [Comments]
# Scraping Recipes Using Node.js, Pipedream, and JSON-LD

A look at building a recipe scraping service using Node and Pipedream.

https://www.raymondcamden.com/2024/06/12/scraping-recipes-using-nodejs-pipedream-and-json-ld

**Comment by robertz** [comment]
That is pretty neat. I will see if I can write a CF version of that :)

**Reply by cfjedimaster** [reply]
Share it when you do. :)
